gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -19,6 +19,7 @@
     <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
 
     <!-- Generated code should not be subjected to checkstyle. -->
+    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
```

Comment:
+1

Do you have a link to the conventions? Only one I could find doesn't reference generated source directly.

Sent from my phone

On May 4, 2016, at 12:48 PM, Chris Purcell <notifications@github.com<mailto:notifications@github.com>> wrote:


In gradle-baseline-java-config/resources/checkstyle/checkstyle-suppressions.xml<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_gradle-2Dbaseline_pull_58-23discussion-5Fr62073631&d=DQMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=aMicnmisMAp8kbnS9GVi4fhmK4gyy4hoQIwVryXsESI&m=hwczNY2VPonvxHSJ3vx8ImHWW-gOBp1zHcw72fMmwIo&s=CMDU1lpewKRR01C2aOlegwDo5W79v1S9tzfce8VpAzA&e=>:

> @@ -19,6 +19,7 @@
>      <suppress files="\.(bdr|eot|gif|gzip|jar|json|otf|png|svg|ttf|woff|zip)$" checks="NewlineAtEndOfFile" />
>
>      <!-- Generated code should not be subjected to checkstyle. -->
> +    <suppress files="[/\\].*[/\\]generated[/\\]" checks="." />
>      <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />


We should stick to maven convention unless there's a really good reason, and maven dictates that src is the stuff that lives in source control.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_gradle-2Dbaseline_pull_58_files_3c58b9b096a92445db3f78aa288a27bbcc11e119-23r62073631&d=DQMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=aMicnmisMAp8kbnS9GVi4fhmK4gyy4hoQIwVryXsESI&m=hwczNY2VPonvxHSJ3vx8ImHWW-gOBp1zHcw72fMmwIo&s=_ojfWwvylDcP_-HAezjYtbIId0guj7tO6w8qT_oH2-A&e=>